src/utilities/core/ApplicationPathHelpers.cxx.in

```diff
@@ -41,443 +41,441 @@
-        // There's no way around it IMHO.
-        // Note: This wouldn't be a concern unless you are launching a symlink that lives in your PATH, and outside the directory where the symlink is
-        // * ./OpenStudioApplicationSymlink will work
-        // * /home/OpenStudioApplicationSymlink will work
-        // * OpenStudioApplicationSymlink itself will not work (if you aren't in the same folder as the symlink)
-
-
-        LOG_FREE(Debug, "ApplicationPathHelpers", "getApplicationPath failed as is, trying in to locate in systemPath");
-        path = findInSystemPath(path);
-        applicationPath = completeAndNormalize(path);
-    }
-    if( !openstudio::filesystem::exists( applicationPath ) || openstudio::filesystem::is_directory( applicationPath ) ) {
-        LOG_FREE_AND_THROW("ApplicationPathHelpers", "getApplicationPath failed!");
-    }
+  LOG_FREE(Debug, "ApplicationPathHelpers", "getApplicationPath, found '" << applicationPath << "'");
 
-    LOG_FREE(Debug, "ApplicationPathHelpers", "getApplicationPath, found '" << applicationPath << "'");
+  return applicationPath;
+}
 
-    return applicationPath;
+openstudio::path getApplicationDirectory() {
+  openstudio::path p = getApplicationPath();
+  if (!p.empty()) {
+    return p.parent_path();
   }
+  return openstudio::path();
+}
 
-  openstudio::path getApplicationDirectory() {
-    openstudio::path p = getApplicationPath();
-    if (!p.empty()){
-      return p.parent_path();
-    }
-    return openstudio::path();
-  }
-
-  bool applicationIsRunningFromBuildDirectory()
-  {
+bool applicationIsRunningFromBuildDirectory() {
 #ifdef _WIN32
-    openstudio::path buildDir = openstudio::toPath(boost::algorithm::to_upper_copy(openstudio::toString(getApplicationBuildDirectory())));
-    openstudio::path runDir = openstudio::toPath(boost::algorithm::to_upper_copy(openstudio::toString(getApplicationDirectory())));
+  openstudio::path buildDir = openstudio::toPath(boost::algorithm::to_upper_copy(openstudio::toString(getApplicationBuildDirectory())));
+  openstudio::path runDir = openstudio::toPath(boost::algorithm::to_upper_copy(openstudio::toString(getApplicationDirectory())));
+  std::string package_dir = "_CPACK_PACKAGES";
 #else
-    openstudio::path buildDir = getApplicationBuildDirectory();
-    openstudio::path runDir = getApplicationDirectory();
+  openstudio::path buildDir = getApplicationBuildDirectory();
+  openstudio::path runDir = getApplicationDirectory();
+  std::string package_dir = "_CPack_Packages";
 #endif
 
-    return pathBeginsWith(buildDir, runDir);
-  }
+  bool isInsideCpackDir = openstudio::toString(runDir).find(package_dir) != std::string::npos;
+  // LOG_FREE(Debug, "ApplicationPathHelpers", "buildDir=" << buildDir << ", runDir=" << runDir << ", isInsideCpackDir=" << isInsideCpackDir);
+  return pathBeginsWith(buildDir, runDir) && !isInsideCpackDir;
```

Comment:
This is the actual change. I wanted to be able to test from the _CPack_Packages staging area, so filter it out when determining if we're running from the build directory or not.